Deployment/BootMenuNoKVM.ps1

```diff
@@ -1,71 +0,0 @@
-﻿# Copyright (c) Microsoft Corporation. All rights reserved.
-# See LICENSE.txt in the project root for license information. 
-
-<# 
- 
```

Comment:
Is there a reason we cannot keep these files as is to support any existing users? @knithinc - do we any dependencies on these scripts?